examples/certificate_rabbitmq.py

```diff
@@ -74,8 +74,8 @@
         print e.oneview_response.get('recommendedActions')
 
 
-# Get an Alias Name
-print('\nGet an Alias Name')
+# Get by Alias Name
+print('\nGet by Alias Name')
 response = oneview_client.certificate_rabbitmq.get_alias_name('default')
```

Comment:
To suit to our other resources, I suggest you just rename get_alias_name to get, since it returns a certificate. All queries are filtered by alias name anyway.